client/app/admin/create-project/create-project.controller.js

```diff
@@ -11,9 +11,27 @@
 
     function createProjectController(mapService, drawService, projectService) {
         var vm = this;
+
+        // Wizard variables
         vm.currentStep = '';
+        vm.taskGrid = false;
+        vm.taskArbitrary = false;
+
+        // AOI variables
         vm.AOIValid = true;
         vm.AOIValidationMessage = '';
+        vm.AOI = null;
+
+        // Creating grid variables
+        vm.sizeOfTasks = 0; 
```

Comment:
If we can solve how to do constants in anglar land, I would use them for the AOI validation fail reasons